src/rulesets/asyncapi/__tests__/asyncapi-payload-examples.ts

```diff
@@ -106,7 +106,7 @@ describe(`Rule '${ruleName}'`, () => {
       expect(results).toEqual([
         expect.objectContaining({
           code: ruleName,
-          message: 'Object should have required property `value`',
+          message: '`1` property should have required property `value`',
```

Comment:
@P0lip This dependency change brings very interesting results messages. I actually never noticed the `Object should have requi...` pattern before.